### ISSUE TITLE
Should we change "curtailment" of solar PV to "overplanting"?

### ISSUE THREAD

**Issue by mabijkerk** [quintel/etmodel#3790]
In a recent meeting, a client notified us that the correct term for not connecting all capacity of solar PV is "overplanting", not "curtailment". Curtailment then only applies shutting down any remaining excess of inflexible production.

In practice I think that curtailment is often used a term for overplanting as well, so it is not a big issue that we call it curtailment in [this section of ETM](https://pro.energytransitionmodel.com/scenario/flexibility/flexibility_net_load/curtailment-solar-pv).

However, there is one reason to rename it. When I open a new II3050 National governance scenario, all types of solar PV have 33% 'ex-ante' curtailment i.e. overplanting which shows up in the solar PV chart like this:
<img width="500" src="https://user-images.githubusercontent.com/67338510/128362334-2d1d8540-ca4b-4156-8997-d96ae8cb88cc.png">

When I then change the solar PV curtailment sliders to 0% it appears in the chart as though there is no curtailment of solar power anymore. This also applies to the solar PV curtailment table:
<img width="500" src="https://user-images.githubusercontent.com/67338510/128361834-5649a1a4-e6e0-44f7-9746-827984170cac.png">

This result is somewhat misleading, because there is still curtailment of volatile producers (which includes solar PV). This is 'ex-post' curtailment and it does not show up in the solar PV curtailment table:
<img width="500" src="https://user-images.githubusercontent.com/67338510/128362098-43d573ce-6d82-4b83-9109-5f8a603cbd85.png">

@redekok what do you think, are you in favour of renaming it?

**Comment by redekok** [comment]
I agree this can be somewhat confusing. Not sure if all users are familiar with the term "overplanting" though. If we clearly explain the difference between "curtailment" and "overplanting" to the users, I think I would be in favour of renaming it. I'm still a bit hesitant if it's worth the effort though. @mabijkerk could you give an indication of the effort this would cost?

**Comment by redekok** [comment]
P.S. I'm also curious what @AlexanderWirtz, @ChaelKruip and @michieldenhaan think about this.

**Comment by AlexanderWirtz** [comment]
Well spotted and interesting to learn a new term. I was not familiar with it. I also suggest we reach out to the iNET group at NBNL who requested this feature for their II3050 work and ask them what their preference is. I would think this is possible within the context of the Flex 2 project, but leave that to you. 

A bit more explanation and documentation cannot hurt.

**Comment by mabijkerk** [comment]
> @mabijkerk could you give an indication of the effort this would cost?

It would entail adjusting the slide that currently has the solar PV curtailment sliders, updating the slider texts and changing the labels in a few charts/tables.

I think it would be interesting to reach out to the iNET group, mainly to determine whether overplanting is a completely new term or whether it is a type of curtailment. If it is a type of curtailment then we can simply rename 'Curtailment of solar PV' to 'Curtailment by overplanting'. If it is a completely different term then it might need some more explanation.

**Comment by mabijkerk** [comment]
This discussion is increasingly relevant given the focus on different types of curtailment. For example, we now also have a form of curtailment that is applied in integrated battery systems of solar PV and inland wind. Different stakeholders use different definitions, such as 'last-resort' curtailment, overplanting (II3050), or direct and indirect curtailment (Ireland).

Not only is this relevant because there is a lot of discussion about it, the current definition for overplanting in the ETM 'curtailment of solar PV' may be confusing for users. This is because the 'normal' form of curtailment, 'last-resort', curtailment is called 'curtailment of volatile producers'. Users may get confused because volatile producers also includes solar PV:

![Screenshot 2022-09-15 at 12 06 54](https://user-images.githubusercontent.com/67338510/190377064-268b8a21-bc7e-4e4c-a032-67060813cac7.png)

I therefore argue that a structural review and update of definitions of curtailment in the ETM is necessary. This would require us to involve relevant stakeholders. Notifying @marliekeverweij

**Comment by mabijkerk** [comment]
Note that it is to be determined whether the slider functionality should be revised as well.
